frontend/src/lib/widgetTheme.ts

```diff
@@ -31,14 +30,21 @@ const borderRadius = SCSS_VARS["$border-radius"]
 const labelFontSize = SCSS_VARS["$font-size-sm"]
 const lineHeightBase = SCSS_VARS["$line-height-base"]
 const lineHeightTight = SCSS_VARS["$line-height-tight"]
+
 const smallTextMargin = SCSS_VARS["$m2-3-font-size-sm"]
 const textMargin = SCSS_VARS["$font-size-sm"]
 const tinyTextMargin = SCSS_VARS["$m1-2-font-size-sm"]
 const spacer = SCSS_VARS.$spacer
 
-export const variables = {
-  borderRadius,
-  spacer,
+// Should use spacers instead of spacer but didn't want to
+// figure out map merges from .scss > SCSS_VARS
+// Using this calculator instead to work with spacer.
+export const spacingCalculator = (spacing?: number): string => {
+  if (!spacing) return spacer
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  const [match, amount, unit] = spacer.match(/(?<amount>\d)(?<unit>.*)/)!
+  const numberAmount: number = parseInt(amount, 10)
+  return match && numberAmount ? `${numberAmount * spacing}${unit}` : spacer
```

Comment:
I want to add more context cause this is a more complicated styling setup. Couple questions to ask:

* Is are the spacing constraints related? I'm assuming so.
* Have we thought of just spacing vars that match our styles. Usually we do something like

```
export const spacing = {
  px: "1px",
  none: "0",
  xxs: "0.375rem",
  xs: "0.25rem",
  sm: "0.5rem",
  md: "0.75rem",
  lg: "1rem",
  xl: "1.25rem",
  xxl: "1.5rem",
  xxxl: "2rem",
}
```

And all of your spacing is related in this way (lg = no args, sm = 0.5, xxxl = 2, md = 0.75, xs = 0.25)

Reply:
Alright. Seems like overkill, but getting rid of SCSS in the change will resolve this more.